### DISCUSSION TITLE
Confirmed providers that worked

### DISCUSSION THREAD

**Discussion by fakemanhk** [fakemanhk/openwrt-jp-ipoe#3] [General]
Until today, the following 2 NTT providers are confirmed OK with the setup instruction:

[OCN](https://service.ocn.ne.jp/hikari/docomo/)
[ぷらら (plala)](https://www.plala.or.jp/p-hikari/)

**Comment by cmspam** [comment]
It may be better to not list the provider which sells the connection, but to list the MAP-E infrastructure provider, since multiple providers use the same MAP-E infrastructure solutions. I think it will work on all of them, but the three which exist that use MAP-E are basically...

1. OCN Virtual Connect (NTT Communications), used by Plala and OCN among others
2. v6 Plus (JPIX/JPNE Infrastructure), used by en Hikari, Otegal Hikari, Sonet, GMO Tokutoku, and many others
3. IPV6 Option (BIGLOBE), used by BIGLOBE, maybe others?

There are others like Arteria and Transix which provide ds-lite solutions, but the MAP-E are provided by only these three, I think.

I am using v6 Plus and can confirm it works, so I guess only necessary to have someone with BIGLOBE / ipv6 option to confirm.

**Comment by samqixin** [comment]
I agree to @cmspam comment that there are 3 vital MAP-E solutions and the network providers are simply using one of them.

I'm using so-net, FTTH maison type, NTT east. It is confirmed to provide v6plus.

**Comment by U-siro** [comment]
ahamo hikari working - with ipTIME A1004ns

**Reply by fakemanhk** [reply]
Is it a new brand?

**Reply by U-siro** [reply]
Yes It’s docomo brand

**Comment by fakemanhk** [comment]
Upgraded my plala to 10G plan, the whole setup is working as expected.

But do note that plala has an option called "GGGG" which claims to have lower latency and more ports available, it's actually based on DS-Lite but not MAP-E, this guide is not applicable.

**Comment by cliffhuang0611** [comment]
@fakemanhk 
I can't seem to get it to work on Softbank 光 even though it runs Docomo lines and uses MAP-E
the MAP-E calculator returns non supported prefix for softbank IPv6 address

**Comment by fakemanhk** [comment]
SoftBank uses NTT's line but they don't use the same system, and I don't
think SoftBank is using MAP-E for IPoE.

**Reply by cliffhuang0611** [reply]
Hmmm, it does say MAP-E in their router system, so I thought this would be worth a shot. Thanks for your hard work investigating all of this anyway!

![image](https://github.com/fakemanhk/openwrt-jp-ipoe/assets/52130319/7c4b8087-1ea8-4e8b-a8f8-ccfd62dfdab5)

![image](https://github.com/fakemanhk/openwrt-jp-ipoe/assets/52130319/ece7e6dd-76fe-48bb-8a63-f0cf76d53a3d)

**Comment by cmspam** [comment]
Softbank doesn't make their implementation public as far as I know. If there were public specs about it, it may be possible to support it.

**Reply by cliffhuang0611** [reply]
from my research that lead to this blog
https://blog.missing233.com/2023/08/13/softbank-hikari-research/
it seems like Softbank uses IPIP, but I am still stuck not able to get the radius packets for peer address

**Comment by fakemanhk** [comment]
> from my research that lead to this blog https://blog.missing233.com/2023/08/13/softbank-hikari-research/ it seems like Softbank uses IPIP, but I am still stuck not able to get the radius packets for peer address

You need to do some packet dump on WAN side to capture it.
Other than SoftBank, even with NTT if you have a plan with dedicated IPv4 address it's the same kind of IPIP tunnel, not MAP-E, probably I should state more clearly at the beginning of the README.
